Project-II-Calculator/calculator/src/components/CalculatorComponents/CalculatorContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import CalculatorDisplay from '../DisplayComponents/CalculatorDisplay';
+import NumberButton from '../ButtonComponents/NumberButton';
+import ActionButton from '../ButtonComponents/ActionButton';
+import './Calculator.css';
+
+export default (props) => {
+  const [numberDisplay, setNumberDisplay] = useState("0");
+
+  const numberPressed = (key) => {
+    numberDisplay === "0" ? setNumberDisplay(key) : setNumberDisplay(numberDisplay + key)
+  }
+
+  const clear = () => {
+    setNumberDisplay("0");
+  }
+
+  return (
+    <div className="calculatorContainer">
+      <CalculatorDisplay numberDisplay={numberDisplay} />
+      <ActionButton onClick={clear} text="clear" />
+      <NumberButton text="÷" red />
+      <NumberButton onClick={numberPressed} text="7" />
```

Comment:
This is completely fine but I don't really like the onClick name you are assigning to the props.
Why don't simply use `<NumberButton numberPressed={numberPressed} text="7" />`